apps/backend/test/parse.spec.ts

```diff
@@ -0,0 +1,147 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { S3 } from 'aws-sdk';
+import { handler } from '../src/lambdas/parseFlatFile';
+import { S3ManagerService } from '../src/s3-manager/s3-manager.service';
+import { S3ManagerModule } from '../src/s3-manager/s3-manager.module';
+import { AwsSdkModule } from 'nest-aws-sdk';
+import * as fs from 'fs';
+import path from 'path';
+
+describe('Parser (e2e)', () => {
+  let service: S3ManagerService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      imports: [
+        S3ManagerModule,
+        AwsSdkModule.forRoot({
+          defaultServiceOptions: {
+            endpoint: 'http://localstack:4566',
+            region: 'ca-central-1',
+            s3ForcePathStyle: true
+          },
+          services: [S3]
+        })
+      ],
+      providers: [S3ManagerService]
+    }).compile();
+    service = module.get<S3ManagerService>(S3ManagerService);
+  });
+
+  it('should be defined', async () => {
+    expect(service).toBeDefined();
+  });
+
+  it('uploads DDF file to parse', async () => {
```

Comment:
This module is independent of DDF or TDI or anything. 

It should test wh a file is uploaded or downloaded.